### ISSUE TITLE
Translate documentation to French

### ISSUE THREAD

**Issue by jemsab** [MLanguage/mlang#48]
In order to be used by French tax office agents, all the project documentation needs to be translated to French

**Comment by denismerigoux** [comment]
Does this also concern variable names and comments in the code?

**Comment by jemsab** [comment]
Not for the moment, I shall get back to you later in the summer when I have more details. As explained elsewhere, this is a PoC of translating all external documentation.

**Comment by mdurero** [comment]
Hello,

I have not been called upon to follow the translation through yet. Nonetheless, as I keep track of my understanding of the tax computation system mostly in French, I wondered how I could incorporate updates to the existing documentation. At the moment most Mlang documentation I know _is_ code comments, using odoc.

So I looked ocamldoc/odoc for a bit and I'm kind of troubled as I found nothing revealing any straightforward use of them in a multilingual context. Are we doomed to build our own ad hoc tools to, for instance, keep a translated version in sync with the generated doc ?

**Comment by Keryan-dev** [comment]
Would I be wrong assuming that developer documentation and user documentation are not one and the same and will not be intended to reach the same people ? If so there is a whole, mostly unwritten, part of the doc that we could decide to have only in French.

Or all the code comment, if that matter. Point is, I don't think we should have to maintain several languages for this project.

**Comment by mdurero** [comment]
You're right. I would argue though, that users and developers are going to be non exclusive groups, that developers from the tax office also expect French text and that part of the two docs can be shared.
Then I also understand that for publication an English version is more commonly understood.
